crates/astria-composer/src/searcher/executor.rs

```diff
@@ -249,6 +294,19 @@ impl Executor {
         Ok(())
     }
 }
+
+enum ExeuctionError {
+    SequencerClientFailure(eyre::Report),
```

Comment:
I recommend not creating new error types that wrap an apex-error like `eyre::Report`. It's also not necessary because AFAIK all of sequencer client's errors are fully typed., so it's just this module that inexplicably tosses an `eyre::Report` in the middle of it. :)